xoinvader/ingame.py

```diff
@@ -13,7 +13,7 @@
 from xoinvader.collision import CollisionManager
 from xoinvader.common import Settings
 from xoinvader.curses_utils import Style
-from xoinvader.enemy_wave import EnemyWave
+from xoinvader.level import Level
```

Comment:
You can sort here imports alphabetically) Move handlers upper, don't hesitate to make code around a little nicer.